store-implementation/hbase-store/src/main/java/uk/gov/gchq/gaffer/hbasestore/operation/hdfs/handler/job/factory/HBaseAddElementsFromHdfsJobFactory.java

```diff
@@ -50,32 +55,44 @@
      * @throws IOException for IO issues
      */
     @Override
-    public Job createJob(final AddElementsFromHdfs operation, final Store store) throws IOException {
-        final JobConf jobConf = createJobConf(operation, store);
+    public List<Job> createJobs(final AddElementsFromHdfs operation, final Store store) throws IOException {
```

Comment:
These implementations are all identical now - is it possible for this method to be pushed up to the JobFactory interface as a default method?